src/index.js

```diff
@@ -26,44 +25,101 @@ class Loader {
   stop () { this.$el.fadeOut('fast'); }
 }
 
+let pagesFlag = true;
+
 // ui elements
+const $html$body = $('html, body');
+const $window = $(window);
+const $document = $(document);
+const $sortButtonList = $('.sortBy li button');
 const $btnList = $('#btnList');
 const $yearField = $('#yearField');
 const $yearTitle = $('#yearTitle');
 const $movieList = $('.movies');
+const $movieCount = $('.movie-count');
+const $arrowToTop = $('.arrowToTop');
+const movieImgSmall = '.movie img';
```

Comment:
Bunch of DOM references and bam string in the middle:
https://www.youtube.com/watch?v=DohRa9lsx0Q